pegasus/sites.v3/advocacy.code.org/public/index.haml

```diff
@@ -68,8 +68,8 @@ video_player: false
       %img{src: "/images/teach-ai-logo.svg", alt: ""}
       .teach-ai-text{style: "text-align: center"}
         %p.heading-sm{style: "color: white;"}
-          =hoc_s(:teach_ai_title)
+          =hoc_s(:ai_promote_literacy)
         %p{style: "color: white; margin-bottom: 0"}
-          =hoc_s(:teach_ai_description)
-      %a.link-button.white.has-external-link{style: "flex: 0 0 auto;", href: "https://teachai.org/toolkit", target: "_blank", rel: "noopener noreferrer"}
```

Comment:
Let's keep this button white (this is a case of knowing when to override the mock 🥲)

Reply:
😭

Reply:
Just checked Figma, and this mock is using an outdated button component that doesn't have a white option. This is too little too late, but I put an [updated button](https://www.figma.com/file/hNBxdgZ40KH4UZFG7ZEVo8/AI-Landing-Page-Updates---Spring-2023?type=design&node-id=3765%3A3010&mode=design&t=2GsNUuLjnVEeKH0r-1) component next to it using the white button.

Reply:
I switched it back to the white button and updated the screenshot! Resolving:)